composer.json

```diff
@@ -21,13 +21,21 @@
       "email": "me@cjmaxik.ru",
       "homepage": "https://cjmaxik.ru",
       "role": "IGA/Dev"
+    },
+    {
+      "name": "Alex Blackham (B3none)",
+      "email": "ablackham2000@gmail.com",
+      "homepage": "https://github.com/b3none",
+      "role": "Dev"
     }
   ],
   "require": {
-    "php": ">=5.6.0",
+    "php": ">=7.1.0",
     "nesbot/carbon": "^1.21",
     "php-http/message": "^1.2",
-    "guzzlehttp/psr7": "^1.3"
+    "guzzlehttp/psr7": "^1.3",
+    "leedavis81/vent": "dev-master",
+    "ext-json": "*"
   },
   "require-dev": {
     "phpunit/phpunit": "^5.3",
```

Comment:
PHPUnit 5.3 is not supported. Use at least v7.

Reply:
Will upgrade, totally missed that one.